src/ecs/commands/runCommandInContainer.ts

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import * as picker from '../../shared/ui/picker'
+import { Window } from '../../shared/vscode/window'
+import { getLogger } from '../../shared/logger'
+import { ChildProcess } from '../../shared/utilities/childProcess'
+import { EcsContainerNode } from '../explorer/ecsContainerNode'
+import { recordEcsRunExecuteCommand } from '../../shared/telemetry/telemetry.gen'
+import { DefaultSettingsConfiguration } from '../../shared/settingsConfiguration'
+import { extensionSettingsPrefix } from '../../shared/constants'
+
+export async function runCommandInContainer(node: EcsContainerNode, window = Window.vscode()): Promise<void> {
+    getLogger().debug('RunCommandInContainer called for: %O', node)
+
+    const verifyAwsCliResponse = await new ChildProcess(true, 'aws').run()
+    if (verifyAwsCliResponse.exitCode !== 0) {
+        window.showErrorMessage(
+            localize('AWS.command.ecs.runCommandInContainer.noCliFound', 'Please install the AWS CLI before proceding')
+        )
+        throw new Error('The AWS CLI is not installed.')
+    }
+
+    const verifySsmPluginResponse = await new ChildProcess(true, 'session-manager-plugin').run()
+    if (verifySsmPluginResponse.exitCode !== 0) {
+        window.showErrorMessage(
+            localize(
+                'AWS.command.ecs.runCommandInContainer.noPluginFound',
+                'Please install the SSM Session Manager plugin before proceding'
+            )
+        )
+        throw new Error('The SSM Session Manager plugin is not installed.')
+    }
```

Comment:
Nit: Can you combine these into a single function? You can also combine the string.

Nit: replace "AWS" in the localized string with `getIdeProperties().company`

Reply:
Moved to a function that will check both and used the suggested call to replace 'AWS'.  Left the strings separate for clarity to the user.